### PR TITLE
CHANGELOG: add release notes for v2.1.12, v2.1.11 and v2.1.10

### PR DESCRIPTION
**Pull request by winkyao**
### What problem does this PR solve? 

Changelogs for v2.1.10 and v2.1.11

@GregoryIan @shenli @jackysp @zz-jason PTAL

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,42 @@
 # TiDB Changelog
 All notable changes to this project will be documented in this file. See also [Release Notes](https://github.com/pingcap/docs/blob/master/releases/rn.md), [TiKV Changelog](https://github.com/tikv/tikv/blob/master/CHANGELOG.md) and [PD Changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
+## [2.1.11] 2019-05-31
+
+* Fix the issue that incorrect schema is used for  `delete from join` [#10595](https://github.com/pingcap/tidb/pull/10595)
```
Comment (jackysp) [inline CHANGELOG.md]:
```suggestion
* Fix the issue that incorrect schema is used for `delete from join` [#10595](https://github.com/pingcap/tidb/pull/10595)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,42 @@
...
+* Fix the issue that the built-in `CONVERT()` may return incorrect field type [#10263](https://github.com/pingcap/tidb/pull/10263)
+* Merge non-overlapped feedback when updating bucket count [#10569](https://github.com/pingcap/tidb/pull/10569)
+* Fix calculation errors  of  `unix_timestamp()-unix_timestamp(now())` [#10491](https://github.com/pingcap/tidb/pull/10491)
```
Comment (jackysp) [inline CHANGELOG.md]:
```suggestion
* Fix calculation errors of  `unix_timestamp()-unix_timestamp(now())` [#10491](https://github.com/pingcap/tidb/pull/10491)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,42 @@
...
+* Fix the incompatibility issue of `period_diff` with MySQL 8.0 [#10501](https://github.com/pingcap/tidb/pull/10501)
+* Skip `Virtual Column` when collecting statistics to avoid exceptions [#10628](https://github.com/pingcap/tidb/pull/10628)
+* Support the `SHOW OPEN TABLES` statement [#10374](https://github.com/pingcap/tidb/pull/10374)
+* Fix the issue that goroutine leak may happen in some cases [#10656](https://github.com/pingcap/tidb/pull/10656)
+* Fix the issue that setting the `tidb_snapshot` variable in some cases may cause incorrect parsing of time format  [#10637](https://github.com/pingcap/tidb/pull/10637)
```
Comment (jackysp) [inline CHANGELOG.md]:
```suggestion
* Fix the issue that setting the `tidb_snapshot` variable in some cases may cause incorrect parsing of time format [#10637](https://github.com/pingcap/tidb/pull/10637)
```

**Comment by winkyao** [comment]
@jackysp PTAL

**Pull request merged**